src/models/thelio-major-r3/repairs.md

```diff
@@ -0,0 +1,483 @@
+
+**Tools required:** None  
+**Time estimate:** 30 seconds  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the front accent strip:
+
+1. Place the system on the edge of the desk so the front side is hanging off of the desk.
+    - The system can alternatively be lifted or tilted so the front of the computer is hovering above the desk.
+2. Slide the accent strip down to unlock it.
+    - The accent can be gripped at the bottom edge.
+3. Pull the accent strip off of the case, starting with the bottom edge.
+4. Place the new accent strip onto the front of the case and slide it up to lock it into place.
+
+![Front accent strip removal](./img/accent-strip-removal.webp)
+
+## Removing the top case:
+
+The top case can be removed to access the internal components.
+
+**Tools required:** Cross-head (Phillips) screwdriver (optional)  
+**Time estimate:** 2 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to remove the top case:
+
+1. Remove the eight outer screws holding the top case onto the machine.
+
+![Top case screws](./img/top-case-screws.webp)
+
+2. Slide the top case up and off of the machine.
+
+## Replacing the GPU:
+
+Thelio Major supports two multi-slot dedicated GPUs:
+
+- The top GPU slot is a PCIe 5.0 x16 slot.
+    - There are four units of clearance in the back of the chassis.
+    - This slot shares bandwidth with M.2 slots 2 and 3 (the bottom two M.2 slots); if either of these M.2 slots are populated, then the top GPU slot will operate at x8 bandwidth.
+- The bottom GPU slot is a PCIe 4.0 x4 slot.
+    - There are two units of clearance in the back of the chassis.
+    - This slot physically fits up to x16 cards, but runs at x4 bandwidth.
+
+Factory configurations only include one GPU.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 15 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the GPU:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case).
+2. Unscrew the four screws holding the side GPU brace in place (two on the front of the case, two on the back.) Remove the brace.
+
+![GPU brace screws](./img/gpu-brace-screws.webp)
+
+3. Unscrew the two back thumbscrews holding the PCIe bracket in place, and remove the PCIe bracket.
+    - The empty PCIe slot covers will come loose, and can also be removed.
+
+![PCIe bracket screws](./img/pcie-bracket.webp)
+
+4. If you're removing a GPU, unplug the GPU power cable (or cables) from the right side of the card. Hold down the latch on the connector while unplugging the cable.
+
+![GPU power connections](./img/gpu-power.webp)
+
+5. Push and hold the latch on the motherboard to free the PCIe connection, then pull the card out of the slot.
```

Comment:
I think it would be good to call out the GPU unlatching button that this motherboard has:
![2023-02-15_17-28-51](https://user-images.githubusercontent.com/13512097/219227028-ea3ef847-1635-44ba-9345-b2b77b1b3ded.png)

Looks like it's still in the shot, so hopefully nothing has to be re-shot.

Reply:
I actually tried pressing that button when I was taking photos and it didn't seem to do anything mechanical, so I assumed it was some kind of power or reset button, lol. I can label it in that image if you're sure that's what it does. (I remembered one of the boards we looked at had one, that's why I tried to check it.)

Reply:
https://www.gigabyte.com/WebPage/922/removePCIE.html - Looks like Gigabyte calls it "EZ-Latch Plus" in the manual, so that confirms it.
![image](https://user-images.githubusercontent.com/7199422/219491623-418b9b7f-fe81-4ca9-991c-3022896f5d40.png)

Reply:
Fixed in https://github.com/system76/tech-docs/pull/162/commits/ee490964fef0222dd33bd8f52c2c8660f6b420bd.